src/main/java/org/ecocean/identity/IBEISIA.java

```diff
@@ -1510,6 +1511,12 @@ public static JSONObject processCallback(String taskID, JSONObject resp, String
                		if (taskParameters == null) taskParameters = new JSONObject();
                		JSONObject tp = new JSONObject();
```

Comment:
i think these 3 lines (above) can be removed/modified. namely: `taskParameters` will _always_ be null, and i think `tp` is never used.